superset/views/sql_lab.py

```diff
@@ -126,11 +127,11 @@ class SavedQueryView(SupersetModelView, DeleteMixin):
 
     show_template = "superset/models/savedquery/show.html"
 
-    def pre_add(self, obj):
-        obj.user = g.user
+    def pre_add(self, item):
```

Comment:
NIT: `item` is probably better, but `obj` is not outrageous, we could add `obj` to Pylint's config of allowed object names.

Reply:
I think that the problem is related with the parent's signature, on FAB the method is `pre_add(self, item)`

Reply:
+1 - the issue is the difference in signature.